includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -640,17 +640,21 @@ public function update_account( $stripe_account_settings ) {
 	 * Get data needed to initialize the OAuth flow
 	 *
 	 * @param string $return_url    - URL to redirect to at the end of the flow.
-	 * @param array  $business_data - data to prefill the form.
+	 * @param array  $business_data - Data to prefill the form.
+	 * @param array  $additional    - Additional data to be sent to the on-boarding flow.
 	 *
 	 * @return array An array containing the url and state fields.
+	 *
+	 * @throws API_Exception Exception thrown on request failure.
 	 */
-	public function get_oauth_data( $return_url, $business_data = [] ) {
+	public function get_oauth_data( $return_url, $business_data = [], $additional = [] ) {
```

Comment:
I also think we can declare the $additional type as `array` without breaking any compatibility.

Reply:
Good plan, I always forget we have the necessary minimum PHP version for this. Please keep prompting when I miss these!

Reply:
Gladly =)